### PR TITLE
Rdk 33900 network thunder plugin document update

### PR DESCRIPTION
**Pull request by sgrand780**
The current network thunder plugin's document is not as per the latest code.
Update the network thunder plugin's document.

Jira ID:https://ccp.sys.comcast.net/browse/RDK-33900
@vijs  could you please review this pull request.

### CODE REVIEW

File: Network/doc/NetworkPlugin.md [hunk 1]
```diff
@@ -949,7 +1000,7 @@ Network interface events:
 | Event | Description |
 | :-------- | :-------- |
 | [onConnectionStatusChanged](#event.onConnectionStatusChanged) | Triggered when a connection is made or lost |
-| [onDefaultInterfaceChanged](#event.onDefaultInterfaceChanged) | Triggered when the default interface changes, regardless if it's from a system operation or through this API |
+| [onDefaultInterfaceChanged](#event.onDefaultInterfaceChanged) | Triggered when the default interface changes, regardless if it's from a system operation or through the `setDefaultInterface` API |
```
Comment (tnykkn) [inline Network/doc/NetworkPlugin.md:1003]: "or through the `setDefaultInterface` method"

File: Network/doc/NetworkPlugin.md [hunk 2]
```diff
@@ -958,6 +1009,14 @@ Network interface events:
 ## *onConnectionStatusChanged <sup>event</sup>*
 
 Triggered when a connection is made or lost.
+ 
+### Methods
+  
+| Method | Description | 
+| :----------- | :----------- |
+| `setDefaultInterface` | Sets the default interface successfully and triggers this event |
```
Comment (tnykkn) [inline Network/doc/NetworkPlugin.md:1017]: The "setDefaultInterface" method may trigger this event (not guaranteed).
For example: If the device's default interface is already WIFI and "setDefaultInterface(WIFI)" is invoked (successfully), there will be no change to the device's default interface and thus no onConnectionStatusChanged or onDefaultInterfaceChanged events generated.

File: Network/doc/NetworkPlugin.md [hunk 3]
```diff
@@ -983,7 +1042,15 @@ Triggered when a connection is made or lost.
...
 ## *onDefaultInterfaceChanged <sup>event</sup>*
 
-Triggered when the default interface changes, regardless if it's from a system operation or through this API.
+Triggered when the default interface changes, regardless if it's from a system operation or through the `setDefaultInterface` API.
+ 
+### Methods
+  
+| Method | Description | 
+| :----------- | :----------- |
+| `setDefaultInterface` | Sets the default interface successfully and triggers this event |
```
[the remark this answers is not in the document]
Comment (tnykkn) [reply Network/doc/NetworkPlugin.md:1051]: Please change the Description column to "Triggers this event if a call to this method causes the device's default interface to change.".

File: Network/doc/NetworkPlugin.md (hunk 1 above)
Comment (sgrand780) [reply Network/doc/NetworkPlugin.md]: Added as suggested.

File: Network/doc/NetworkPlugin.md (hunk 2 above)
Comment (sgrand780) [reply Network/doc/NetworkPlugin.md]: Updated as per suggestion.

File: Network/doc/NetworkPlugin.md [hunk 4]
```diff
@@ -1009,7 +1076,16 @@ Triggered when the default interface changes, regardless if it's from a system o
...
 ## *onInterfaceStatusChanged <sup>event</sup>*
 
-Triggered when an interface becomes enabled or disabled.
+Triggered when an interface becomes enabled or disabled. This event is triggered during below methods call sequence.
+ 
+### Methods
+  
+| Method | Description | 
+| :----------- | :----------- |
+| `setDefaultInterface` | Default interface changed successfully and triggers this event |
```
[the remark this answers is not in the document]
Comment (sgrand780) [reply Network/doc/NetworkPlugin.md]: Updated as per suggestion.

**Comment by sgrand780** [comment]
@tnykkn 
I have addressed and updated the .md and .json files. 
Please review once.

**Pull request closed without merging**